nncf/quantization/algorithms/accuracy_control/openvino_backend.py

```diff
@@ -101,5 +124,7 @@ def get_model_size(model: ov.Model) -> int:
     # Preparation of model
 
     @staticmethod
-    def prepare_for_inference(model: ov.Model) -> ov.CompiledModel:
-        return ov.compile_model(model)
+    def prepare_for_inference(model: ov.Model) -> OVPreparedModel:
```

Comment:
This method was removed from `AccuracyControlAlgoBackend` class

Reply:
removed